testsuite/features/step_definitions/common_steps.rb

```diff
@@ -90,7 +90,13 @@
 end
 
 When(/^I check the MAC address value$/) do
-  get_mac_address = 'cat /sys/class/net/eth0/address'
+  get_system_id_like = "awk -F= '/^ID_LIKE/{print $2}' /etc/os-release"
+  id_like, _local, _remote, _code = $client.test_and_store_results_together(get_system_id_like, 'root', 600)
+  if id_like == 'debian'
+    get_mac_address = 'cat /sys/class/net/ens3/address'
+  else
+    get_mac_address = 'cat /sys/class/net/eth0/address'
+  end
   mac_address, _local, _remote, _code = $client.test_and_store_results_together(get_mac_address, 'root', 600)
```

Comment:
Here, make it generic by using `node` instead of `$client`.

We are a bit coding for the future, and YAGNI, but okay :smile_cat: .

Could I please also ask you to document this new step in `documentation/cucumber_steps.md`? When we refactor, we should also profit of the occasion to document.

Reply:
That step is not defined in the doc, where you prefer to add in Operating System or Texts sections.

Reply:
Yes, we miss a lot of stuff in documentation.

System should be fine.